dom.bs

```diff
@@ -1065,6 +1065,15 @@ seen from the definition above, an <a>event listener</a> is a more broad concept
 method, when invoked, must run these steps:
 
 <ol>
+ <li>
+  <p>If the <a>context object</a>'s global object is a {{ServiceWorkerGlobalScope}} object and its
+  associated <a>service worker</a>'s <a>script resource</a>'s <a>has ever been evaluated flag</a> is
+  set, <a>throw</a> a <code>TypeError</code>. [[!SW]]
+
+  <p class="note no-backref">To optimize storing the event types allowed for the service worker and
+  to avoid non-deterministic additions of the event listeners, the user agent allows the invocation
```

Comment:
s/additions of/changes to/
s/the user agent allows the invocation of the method/invocation of the method is allowed/